src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.LocalFunctions.cs

```diff
@@ -191,20 +191,20 @@ protected override LocalFunctionState LocalFunctionStart(LocalFunctionState star
                 currentState.InvertedCapturedMask.Invert();
             }
             // Filter the modified state variables to only captured variables
-            stateAtReturn.Assigned.IntersectWith(currentState.CapturedMask);
+            stateAtReturn.Assigned.IntersectWith(in currentState.CapturedMask);
```

Comment:
Were all the changes done manually or through some analyzer/code fixer?

Reply:
The changes were automated through a code fix.